api/migrations/20190305160606_add_sf_type_and_version.sql

```diff
@@ -0,0 +1,16 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+-- +goose StatementBegin
+ALTER TABLE accounts ADD COLUMN sf_type VARCHAR(25) NOT NULL DEFAULT 'SF86';
+ALTER TABLE accounts ALTER COLUMN sf_type DROP DEFAULT;
+ALTER TABLE accounts ADD COLUMN sf_version VARCHAR(25) NOT NULL DEFAULT '2016-11';
```

Comment:
Just a comment, not asking for a change. I'm not a big fan of varchars with fixed limits in the schema, though that is in keeping with the rest of the eApp schema. See:
https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_varchar.28n.29_by_default

Reply:
I can switch this to text, its an old habit that probably needs to die.